portal/pages/api/auth/[...nextauth].js

```diff
@@ -4,18 +4,16 @@ import axios from "axios";
 import fs from "fs";
 import pathFile from "path";
 import jwt from "jsonwebtoken";
-const CryptoJS = require("crypto-js");
 
 const verifyOrCreate = async (token, refreshToken) => {
   const cert = fs.readFileSync(pathFile.resolve("", "./jwt.pem"));
-  let verify = false;
   return jwt.verify(
     token,
     cert,
     { algorithms: ["RS256"] },
-    async function (err, payload) {
+    async function (err) {
       if (err) {
-        try {
+        // try {
```

Comment:
Why we removed try block?

Reply:
eslint was showing errors like unnecessary try catch block.